### PR TITLE
fix doc for sql support

### PR DESCRIPTION
**Pull request by jingshanglu**

### CODE REVIEW

File: docs/document/content/features/sharding/use-norms/sql.en.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ It is inevitably to have some unlisted SQLs, welcome to supplement for that. We
 
 ### Route to multiple data nodes
 
-Fully support DML, DDL, DCL, TCL and some DAL. Support pagination, DISTINCT, ORDER BY, GROUP BY, aggregation and JOIN (does not support cross-database relevance). Here is an example of a most complex kind of DML:
+Partial support DML, DDL, DCL, TCL and some DAL. Support pagination, DISTINCT, ORDER BY, GROUP BY, aggregation and JOIN (does not support cross-database relevance). Here is an example of a most complex kind of DML:
```
Comment (tristaZero) [inline docs/document/content/features/sharding/use-norms/sql.en.md:20]: -> Partially

**Review by tristaZero — CHANGES_REQUESTED** [review]

**Review by terrymanu — CHANGES_REQUESTED** [review]
The document is not clear, we need to explain `Partially`'s details

File: docs/document/content/features/sharding/use-norms/sql.cn.md [hunk 2]
```diff
@@ -49,7 +49,12 @@ tbl_name [AS] alias] [index_hint_list]
 
 ### 路由至多数据节点
 
-不支持CASE WHEN、HAVING、UNION (ALL)，有限支持子查询。
+部分支持CASE WHEN
+* `CASE WHEN` 中包含子查询不支持
+* `CASE WHEN` 中使用逻辑表名不支持（请使用表别名）
+不支持 HAVING、UNION (ALL)
+部分支持子查询
+* 子查询中没有分片键条件不支持，参考[子查询支持详情](https://github.com/apache/shardingsphere/issues/6497)
```
Comment (terrymanu) [inline docs/document/content/features/sharding/use-norms/sql.cn.md:57]: Can we summary the issue content as document?

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: docs/document/content/features/sharding/use-norms/sql.cn.md [hunk 3]
```diff
@@ -49,20 +49,25 @@ tbl_name [AS] alias] [index_hint_list]
 
 ### 路由至多数据节点
 
-不支持CASE WHEN、HAVING、UNION (ALL)，有限支持子查询。
+部分支持CASE WHEN
+* `CASE WHEN` 中包含子查询不支持
+* `CASE WHEN` 中使用逻辑表名不支持（请使用表别名）
+不支持 HAVING、UNION (ALL)
+部分支持子查询
+* 存在子查询，同时sql中没有分片键条件不支持，如果子查询中有分片键，需要和父查询中分片键条件一致
```
Comment (strongduanmu) [inline docs/document/content/features/sharding/use-norms/sql.cn.md:57]: I feel this expression a bit difficult to understand, how about this expression —— "子查询中必须包含分片键，且分片键必须和父查询中的分片键一致". 😊

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: docs/document/content/features/sharding/use-norms/sql.cn.md (hunk 3 above)
Comment (jingshanglu) [reply docs/document/content/features/sharding/use-norms/sql.cn.md:57]: Ok

**Review by strongduanmu — APPROVED** [review]

**Review by terrymanu — APPROVED** [review]

**Review by tristaZero — APPROVED** [review]

**Pull request merged**